src/components/image/index.tsx

```diff
@@ -183,6 +185,13 @@ class Image extends PureComponent<Props, State> {
     return this.getVerifiedSource(source);
   }
 
+  getImageStyle = () => {
```

Comment:
I think we should wrap this with `useMemo`, also `this.shouldUseImageBackground()` should be memoized since we are using it's value more than once and store it's value into variable.
WDYT ?

Reply:
WDYM? 
First of all, notice this is a class component, we can't use `useMemo` here. 
Also there's no point in memoizing, notice I return this `return borderRadius ? [{borderRadius}, imageStyle] : imageStyle;` so if a `borderRadius` was passed the style will be dynamic anyway and if it wasn't passed, I am just passing the `imageStyle` prop that the user has passed. 

as for `this.shouldUseImageBackground()` we don't store its value anywhere.. it's just a util method. where do we store its value in a variable?

Reply:
I had some mistake here my bad.
About the `getImageStyle` you are right since the return statement will return `imageStyle`.